examples/react-native/lingui.config.js

```diff
@@ -0,0 +1,12 @@
+/** @type {import('@lingui/conf').LinguiConfig} */
+module.exports = {
+	locales: ["en", "cs"],
```

Comment:
I just noticed that some files in the sample project have a TABs indentation. Can we change them to a common style?